### ISSUE TITLE
Capture Region menu option won't click

### ISSUE THREAD

**Issue by bigbarryq** [Aminadav/screenshot-extension#52]
Windows 8.1 64bit, Chrome Version 54.0.2840.99 m

Can't click on Capture Region option. Can highlight the text but can't click. Works on Windows 7 64 bit with same Chrome.

**Comment by klajdi369** [comment]
I think I have the same problem. Whenever i want to select a region, and when I press "Capture region of screen" nothing happends.

**Comment by sljohnson32** [comment]
I have the same issue right now on MacBook with latest Chrome.

**Comment by Aminadav** [comment]
It should be fixed. Please test the last version.

**Comment by sljohnson32** [comment]
I just added it back to Chrome and still go no action when clicking the capture region option.  

I am on a Mac with Chrome version 64.0.3282.167 (Official Build) (64-bit)

**Comment by Ezrway** [comment]
I tried to capture a Whole Page Screenshot of this site: https://support.google.com/wifi/answer/7183148?hl=en&ref_topic=6243113 and the extension produced a blank PDF file.

I tried to attach the file but I kept getting a "Something went really wrong, and we can't process that file." (in red).  I used the "Try again." option with the same result.  I named the file so there were no spaces in the file name just in case that mattered but it didn't help.
